main.js

```diff
@@ -11,3 +11,140 @@ function closemenu() {
 //  binding the event to HTML
 menu.addEventListener('click', openmenu);
 menu.addEventListener('click', closemenu);
+
+const workInfo = [
+  {
+    Id: '1',
+    name: 'Tonic',
```

Comment:
- [ ] great work. please kindly ensure that your applicatyilon follows the figma design. i cant seem to see yiour seelive/seesource buttons in the desktop screen. you can reduce the popup size so all fits
- [ ] kindly ensure the close icon is also visible in the desktop version
- [ optional] please like in the figma version, kindly make the background a little blury when the popup comes up. this will push all the users attention to the current popup project.

yours
![Screenshot (694)](https://user-images.githubusercontent.com/65298048/209297927-7b7aa77b-1d94-475b-8c3c-35c06548a9ff.png)
figma
![Screenshot (691)](https://user-images.githubusercontent.com/65298048/209298074-66de6aea-ac76-4281-9651-c076492e4da8.png)